types/node/crypto.d.ts

```diff
@@ -196,6 +196,43 @@ declare module 'crypto' {
         cipher?: string;
         passphrase?: string | Buffer;
     }
+    interface JwkKeyExportOptions {
+        format: 'jwk';
+    }
+    interface JsonWebKey {
+        crv?: string;
+        d?: string;
+        dp?: string;
+        dq?: string;
+        e?: string;
+        k?: string;
+        kty?: string;
+        n?: string;
+        p?: string;
+        q?: string;
+        qi?: string;
+        x?: string;
+        y?: string;
+    }
```

Comment:
We should probably add an indexable unknown/any here so that JWKs with extra properties can still be passed in.

e.g.

```suggestion
    interface JsonWebKey {
        crv?: string;
        d?: string;
        dp?: string;
        dq?: string;
        e?: string;
        k?: string;
        kty?: string;
        n?: string;
        p?: string;
        q?: string;
        qi?: string;
        x?: string;
        y?: string;
        [key: string]: any;
    }
```

Reply:
Turns out the cherry pick duplicated a few things, applied suggestion in existing interfaces.